README.md

```diff
@@ -26,6 +26,10 @@ Because of the new minimum MSBuild version requirement of these new SDKs, it wil
 
 ## What's new in 1.23.1
 -   Register FixAll commands for disposal ([#3984](https://github.com/OmniSharp/omnisharp-vscode/issues/3984), PR: [#3985](https://github.com/OmniSharp/omnisharp-vscode/pull/3985))
+-   Register FixAll commands for disposal ([#3984](https://github.com/OmniSharp/omnisharp-vscode/issues/3984), PR: [#3985](https://github.com/OmniSharp/omnisharp-vscode/pull/3985))
+-   Include version matched target files with minimal MSBuild (PR: [omnisharp-roslyn#1895](https://github.com/OmniSharp/omnisharp-roslyn/pull/1895))
+-   Fix lack of trailing italics in quickinfo (PR: [omnisharp-roslyn#1894](https://github.com/OmniSharp/omnisharp-roslyn/pull/1894))
+-   Start moving omnisharp to directly using Roslyn's completion service (PR: [omnisharp-roslyn#1877](https://github.com/OmniSharp/omnisharp-roslyn/pull/1877))
```

Comment:
I think this is not yet relevant to the C# extension

Reply:
Good point. =) Looks like new completion will wait on https://github.com/dotnet/aspnetcore/issues/25008https://github.com/dotnet/aspnetcore/issues/2500